src/ExportGenerators.mss

```diff
@@ -704,6 +716,16 @@ function GenerateLayers (staffnum, measurenum) {
             case('Text')
             {
                 mobj = ConvertText(bobj);
+                if (mobj != null)
+                {
+                    //Try to get note at position of bracket and put id
+                    obj = GetNoteObjectAtPosition(bobj);
+
+                    if (obj != null)
+                    {
+                        libmei.AddAttribute(text, 'startid', '#' & obj._id);
```

Comment:
@annplaksin `text` should be `mobj`. Can you change this and squash it into commit dd3ba30?

Reply:
Is done...